src/handbook/peopleops/job-descriptions/account_executive.md

```diff
@@ -0,0 +1,58 @@
+We are seeking a highly motivated, driven, and self-sufficient Account Executive (a "Sales Explorer") to join our team and drive growth in the local region. In this early-stage growth opportunity, you will be responsible for identifying and closing new business opportunities, building strong customer relationships, and contributing to the overall success of FlowFuse. This role requires someone who can build pipeline, refine messaging, and optimize our sales motion based on real conversations, demonstrating a willingness to experiment and a comfort level with ambiguity.
+
+* Expand FlowFuse's customer base and revenue in the regional market, targeting key accounts using an Account-Based Marketing (ABM) approach.
+* Identify and engage new prospects, and drive deals from initial contact to close.
+* Test and refine our sales motion to optimize effectiveness in the regional market.
+* Generate pipeline through proactive prospecting and creative outreach, going beyond traditional cold emails.
+* Effectively communicate the value proposition of FlowFuse to both technical and business stakeholders.
+* Collaborate cross-functionally with marketing, product, and customer success to align on go-to-market strategy.
+* Gather and communicate market feedback to inform product development and sales enablement.
+
+## Skills
+* Startup Sales DNA: Experience working in an early-stage company (or have the mindset to thrive in one). Embraces ambiguity, iteration, and autonomy.
+* Full-Cycle Sales Experience: Proven experience selling B2B SaaS or enterprise technology and navigating complex sales cycles with multiple stakeholders.
+* Curiosity & Adaptability: A passion for testing new approaches, learning from data, and evolving sales strategies to match market needs.
+* Creative Prospecting Skills: Ability to go beyond cold emails and find innovative ways to connect with buyers.
+* Communication Skills: Excellent verbal and written communication skills with the ability to present complex information clearly.   
+* Technical Acumen: Comfortable understanding and explaining technical solutions such as Node-RED, industrial automation, IoT, or DevOps tooling.
+* Cultural Awareness: Strong understanding of cultural nuances and business practices within the region.
+* Familiarity with AI tools to enhance sales processes is a plus.
+* A strong attitude match with the existing team is critical, as is experience managing both inbound leads and creating outbound strategies.
+
+## 90-Day Plan:
+
+* Week 1-4:
+   * Immerse yourself in FlowFuse’s product, strategy, and market with a focus on enterprise-readiness, applications on the edge, and enhanced integration capabilities.
+   * Familiarize yourself with the Node-RED ecosystem, competitive landscape, and specific regional market dynamics.
+   * Meet with key stakeholders across sales, marketing, product, and customer success.
+   * Review existing sales materials and processes, and identify areas for improvement.
```

Comment:
I would say they start selling week 2 onwards. 7 days to review other calls, and learn the pitch. Bring an experienced seller along, and let's go. This is quite passive for 4 weeks IMO.